modules/identity-provider-oidc-supported.adoc

```diff
@@ -5,7 +5,7 @@
 [id="identity-provider-oidc-supported_{context}"]
 = Supported OIDC providers
 
-The following OpenID Connect (OIDC) providers are tested and supported with {product-title}:
+OpenID Connect (OIDC) providers that comply with OpenID standards are supported with {product-title}. The following list outlines supported OpenID Connect (OIDC) providers that are tested with {product-title}:
```

Comment:
I'm not sure if the use of "**supported**" in "OpenID Connect (OIDC) providers that comply with OpenID standards are **supported** with {product-title}" is good. I doubt. Per https://url.corp.redhat.com/d603b52 , only tested are supported:
```
"Tested: Where this term is used, Red Hat has explicitly tested OpenShift (for the denoted minor
version[s]) with ... Where this term is seen customers should expect the documented features
connected/related to this integration to work, and where they do not, users should open a
support case so that the issue can be investigated and addressed."
```

Reply:
Thanks, @xingxingxia . I updated the content. Please let me know your thoughts.